modules/porous_flow/src/materials/PorousFlowPermeabilityConst.C

```diff
@@ -0,0 +1,32 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowPermeabilityConst.h"
+
+template<>
+InputParameters validParams<PorousFlowPermeabilityConst>()
+{
+  InputParameters params = validParams<Material>();
+
```

Comment:
Extra line not needed.  Also, you have to have validParams<PorousFlowPermeabilityUnity>();